composer.lock

```diff
@@ -3037,6 +3112,53 @@
             "description": "A small library for converting tokenized PHP source code into XML and potentially other formats",
             "time": "2017-04-07T12:08:54+00:00"
         },
+        {
+            "name": "vkartaviy/retry",
```

Comment:
tady máš furt to retry :(

Reply:
už je pryč + jsem nahrál všechny required balíčky do composeru